db/rdbms/src/main/resources/db/changelog/rdbms-support/changesets/create_engine.xml

```diff
@@ -23,10 +23,16 @@
       </column>
       <column name="PROCESS_DEFINITION_ID" type="VARCHAR(200)" />
       <column name="NAME" type="VARCHAR(200)" />
+      <column name="RESOURCE_NAME" type="VARCHAR(400)" />
```

Comment:
Hm in general we should maybe sync the size of the varchars with c7. For Resource_Name, there they use 4000 .... 

```
create table ACT_RE_PROCDEF (
    ID_ nvarchar(64) not null,
    REV_ int,
    CATEGORY_ nvarchar(255),
    NAME_ nvarchar(255),
    KEY_ nvarchar(255) not null,
    VERSION_ int not null,
    DEPLOYMENT_ID_ nvarchar(64),
    RESOURCE_NAME_ nvarchar(4000),
    DGRM_RESOURCE_NAME_ nvarchar(4000),
    HAS_START_FORM_KEY_ tinyint,
    SUSPENSION_STATE_ tinyint,
    TENANT_ID_ nvarchar(64),
    VERSION_TAG_ nvarchar(64),
    HISTORY_TTL_ int,
    STARTABLE_ bit NOT NULL default 1,
    primary key (ID_)
);
```